### DISCUSSION TITLE
Close : Members Browser Site Page

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#208] [Q&A] [answered]
2 questions about the member manager

In the Site tab, what should be done? 
Class
General registration number

**Comment by meteos77** [comment]
suggestion:
next to the Maximum number of books that can be borrowed have a new option that would specify the 
"Default borrowing time in days".

this should influence the date at the time of reservation
currently it is 1 day, if this parameter was adjustable great

**Reply by meteos77** [reply]
I had completely missed this parameter in the database enumeration.

now it's impossible to miss :-)
thanks a lot

**Comment by textbrowser** [comment]
The Site page contains site-specific information. Future and present.

**Comment by meteos77** [comment]
for the moment I don't use it but what is the use of the field : General registration number ?

**Comment by textbrowser** [comment, marked as answer]
Perhaps the patron is part of a committee. Or a volunteer group. It was requested years ago.
